docs/src/manifolds/power.md

```diff
@@ -1,4 +1,4 @@
-# Product manifold
+# Power manifold
```

Comment:
Saw this one just, too, in the published docs.

Reply:
So it looks correct now :slightly_smiling_face: